modules/sql-engine/src/main/java/org/apache/ignite/internal/sql/engine/util/Commons.java

```diff
@@ -861,4 +864,46 @@ public static SqlQueryType getQueryType(SqlNode sqlNode) {
                 return null;
         }
     }
+
+    /** Returns the minimum unscaled value of a numeric type.
+     *
+     * @param type a numeric type
+     */
+    public static long getMinValue(RelDataType type) {
+        SqlTypeName typeName = type.getSqlTypeName();
+        switch (typeName) {
+            case TINYINT:
+                return Byte.MIN_VALUE;
+            case SMALLINT:
+                return Short.MIN_VALUE;
+            case INTEGER:
+                return Integer.MIN_VALUE;
+            case BIGINT:
```

Comment:
Is BIGINT scaled? or `Long.MIN_VALUE` is missed here?

Reply:
I just copy-paste this methods from calcite, cause i found that they not used anywhere and probably can be removed .. Plz explain what i need to check here ?

Reply:
`type.getPrecision()` is 19 for BIGINT
`NumberUtil.getMinUnscaled(type.getPrecision())` creates `BigInteger.valueOf(Long.MIN_VALUE)` for param value of 19, then converts to long.

Seems, it is safe to return Long.MIN_VALUE here.
Same is true for max_value.